aries-site/src/data/structures/components.js

```diff
@@ -44,14 +44,14 @@ export const components = [
       background: 'background-front',
     },
     relatedContent: ['Button', 'Typography', 'Forms'],
-    tags: ['hyperlinks', 'links', 'linking', 'navigation'],
+    tags: ['hyperlinks', 'links', 'linking', 'navigation', 'anchors', 'buttons', 'accessibility standards for links', 'link labels'],
   },
   {
     name: 'Button',
     category: 'Controls',
-    description: 'Buttons are used to indicate actions that can be performed.',
+    description: 'Buttons are graphic elements that indicate to users that actions can be performed.',
```

Comment:
Pausing on the term "graphic" since I feel like graphical elements on web typically refer to images, svgs, decorators, etc.

Reply:
agree I dont think `graphic` is good here

Reply:
should we revert?